integration/copy_test.go

```diff
@@ -151,6 +151,43 @@ func (s *CopySuite) TestCopySimple(c *check.C) {
 	c.Assert(err, check.IsNil)
 }
 
+// Make sure that docker-archive is identical to docker.
```

Comment:
This is not really describing the test… what would “a local file being identical to a remote repository” (or is it “to a daemon”) mean? And docker does nothing with OCI…

I guess, a test is a test is a test, and it _is_ useful even if not explained, but knowing what is being tested could make it easier in the future to figure out whether a failure is a bug in the test or in the software being tested.